### PR TITLE
Add other Azure Clouds

### PR DESCRIPTION
**Pull request by CKGrafico**
In Azure exist more than one Cloud, we have:
Azure Cloud
Azure China Cloud
Azure US Government
Azure German Cloud
Azure Stack

With this PR the plugin will be ready to be extended to other clouds, and also I've configured the China cloud (because I need it asap)

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -27,10 +38,16 @@ module.exports = {
     container: {
       label: 'Container',
       type: 'text'
+    },
+    azureType: {
+      label: 'Azure Type',
+      type: 'enum',
+      values: [azureTypes.base.label, azureTypes.china.label]
```
Comment (sylvaingirardbe) [inline lib/index.js]: Why aren't you using Object.values here?

Comment (CKGrafico) [reply lib/index.js]: Do you prefer something like this?
`Object.values(azureTypes).map(x => x.label)`

done

**Comment by CKGrafico** [comment]
Thanks! please if you can tell us when you have the new npm version ready :)

**Comment by sylvaingirardbe** [comment]
> Thanks! please if you can tell us when you have the new npm version ready :)

Done! v0.1.0 ready for download.

**Pull request merged**